scaling_performance/optimizing_on_glusterfs_storage.adoc

```diff
@@ -0,0 +1,83 @@
+[[scaling-performance-optimizing-on-glusterfs-storage]]
+= Optimizing On GlusterFS Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+toc::[]
+include::_snippets/glusterfs.adoc[]
+
+[[Guidance-For-Databases]]
+== {gluster-native} Guidance for Databases
+When you use {gluster-native} for applications, follow the guidance and best practices provided in this topic so 
+that you can make informed choices between gluster-block and GlusterFS modes based on your type of workload.
+
+[[Tested-Applciations]]
+== Tested Applications 
+In {product-title} 3.10, extensive testing was done with these (no)SQL databases:
+
+-  Postgresql SQL v9.6 
+-  Mongodb noSQL v3.2
+
+The storage for these databases originated from a {gluster-native} storage cluster.
+
+For Postgresql SQL benchmarking link:https://www.postgresql.org/docs/10/static/pgbench.html[pgbench] was used for database benchmarking.
+For MongoDB noSQL benchmarking YCSB link:https://github.com/brianfrankcooper/YCSB/tree/master/mongodb[Yahoo! Cloud Serving Benchmark] was used for benchmarking and link:https://github.com/brianfrankcooper/YCSB/tree/master/workloads[workloada,workloadb,workloadf] were tested
+
+[[Support-Matrix]]
+== Support Matrix 
+
+.Table Title - GlusterFS
+|===
+|Database | Storage backend: *GlusterFS* |Turn *off* Performance Translators |Turn *on* Performance Translators
+|Postgresql SQL
+|Yes
+a| 
+- performance.stat-prefetch 
+- performance.read-ahead  
+- performance.write-behind 
+- performance.readdir-ahead 
+- performance.io-cache 
+- performance.quick-read 
+- performance.open-behind 
+a| 
+- performance.strict-o-direct
+
+|MongoDB noSQL
+|Yes
+a|
+- performance.stat-prefetch 
+- performance.read-ahead  
+- performance.write-behind 
+- performance.readdir-ahead 
+- performance.io-cache 
+- performance.quick-read 
+- performance.open-behind 
+a| 
+- performance.strict-o-direct 
+|===
+
+
+.Table Title - gluster-block
+|===
+|Database | Storage backend: *gluster-block* 
+|Postgresql
+|Yes
+
+
+|MongoDB  
+|Yes
+
+|===
+
+The performance translators for the GlusterFS, as mentioned above, are already part of the database profile delivered with 
+the latest {gluster-native} images.
```

Comment:
Maybe give an example of the gluster command, eg, `gluster volume set <gluster_volume_name> performance.stat-prefetch off` to show how to turn on/off a translator. If a user wants to follow the suggestions in this page, he/she can do it with the command.

Reply:
I would prefer if we omit that and leave up to use to decide how to update it , one will use `gluster volume set ...` , some others will go with `ansible` , or maybe integrate all of them in `volumeoptions` in storage class. As I wrote, we will have these integrated in `db-profile` and they serve here only as information what changed. @jarrpa @hongkailiu comments?

Reply:
Got it. A new `sc` is a cool idea.

Reply:
ACK what @ekuric said.